cobigen-setup/index.asciidoc

```diff
@@ -0,0 +1,23 @@
+= Tutorial for Cobigen Setup
+====
+CobiGen is a generic incremental code generator. It allows you to build Java CRUD application based on the devonfw architecture including all software layers. You can generate all necessary classes and services (DAOs, Transfer Objects, simple CRUD use cases with REST services) of the application from one single entity class.
+
+CobiGen provides Maven and Eclipse integrations. But there is also a command line interface (CLI) which enables the generation of code using only few commands.
+
+More information about CobiGen on https://devonfw.com/website/pages/docs/master-cobigen.asciidoc.html
+====
+
+[step]
+--
+restoreDevonfwIde(["java","mvn"])
+--
+
+For this tutorial an installation of the devonfw IDE is required, which was already set up for you.
+To begin you need to install CobiGen and create a Java Project.
+[step]
+== Install CobiGen and create Java Project
+--
+installCobiGen()
+--
+
+====
```

Comment:
This is a syntax error. See https://github.com/devonfw-tutorials/tutorials/wiki/Development